util/test/checkChplInstall

```diff
@@ -25,16 +25,35 @@
 # To use something other than $HOME/.chpl, set the CHPL_CHECK_INSTALL_DIR
 # environment variable.
 
+function log_info()
+{
+    local msg=$@
+    echo "[INFO] ${msg}"
+}
 
-# Base name for test job in $CHPL_HOME/test/release/examples
-TEST_JOB_BASENAME=hello6-taskpar-dist
+function log_warning()
+{
+    local msg=$@
+    echo "[WARNING] ${msg}"
+}
 
-err() {
-    echo "error: $@" >&2
+function log_error()
+{
+    local msg=$@
+    echo "[ERROR] ${msg}" >&2
 }
 
+function log_success()
+{
+    local msg=$@
+    echo "[SUCCESS] ${msg}"
```

Comment:
I'm tempted to suggest using `INFO`, `WARN`, `FAIL`, and `PASS` so that the actual message are in line. 

I experimented with adding spaces to get the right alignment, but it looked a little off.  Up to you if you want to change, I'm happy either way